internal/service/fleet.go

```diff
@@ -41,9 +41,6 @@ func (h *ServiceHandler) CreateFleet(ctx context.Context, request server.CreateF
 	if errs := request.Body.Validate(); len(errs) > 0 {
 		return server.CreateFleet400JSONResponse{Message: errors.Join(errs...).Error()}, nil
 	}
-	if err := validateDiscriminators(request.Body); err != nil {
```

Comment:
Should you also remove the other call in this file to the function, and delete the function?

Reply:
Right, `validateDiscriminators` is now unnecessary also for Replace / PUT.